src/Control/Email/Email.php

```diff
@@ -706,30 +719,39 @@ public function setBody($body)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::html()
+     *
      * @return $this
      */
     public function invalidateBody()
     {
+        Deprecation::notice('4.12.0', 'Will be replaced with Symfony\Component\Mime\Email::html()');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Use html() instead');
```

Reply:
This feels more like a convenience method - what's the harm in keeping it and implementing as `$this->html(null);` (or whatever the equivalent would be) instead?

Reply:
`$this->html(null)` isn't exactly inconvenient

Reply:
I would argue it's in the same vain as some of the aliases we're keeping - but if we are deprecating this please remove the FQCN from the deprecation notice message. The method will be on _this_ class.

Reply:
Updated